app/controllers/discourse_automation/admin_discourse_automation_triggerables_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module DiscourseAutomation
+  class AdminDiscourseAutomationTriggerablesController < ::ApplicationController
+    def index
+      if params[:automation_id].present?
+        automation = DiscourseAutomation::Automation.find(params[:automation_id])
```

Comment:
What happens if the automation isn't found?

Reply:
ActiveRecord::RecordNotFound which should result in a 404